packages/cli/src/cmds/configValidator/validate-config.js

```diff
@@ -0,0 +1,57 @@
+const fs = require('fs');
+const { join } = require('path');
+const process = require('process');
+
+const yaml = require('js-yaml');
+
+const { verbose } = require('../../utils');
+const validator = require('../../service/config/validator');
+
+exports.command = 'validate-config';
+exports.describe = 'Validate AppMap configuration';
+
+exports.builder = (args) => {
+  /* TODO: have the user specify the project type, get the scheme file from the agent installation
+  args.positional('schema file', {
+    describe: 'File containing config schema for this language',
+  });
+  */
+  args.option('dir', {
+    describe: 'directory in which to perform validation',
+    default: '.',
+    alias: 'd',
+  });
+  return args.strict();
+};
+
+const cmdHandler = async (
+  schema,
+  config,
+  configFile,
+  svc = validator,
+  logger = console
+) => {
+  const result = svc.validateConfig(schema, config);
+  if (result.valid) {
+    logger.log(`\n${configFile} is valid.\n`);
+    return 0;
+  }
+
+  logger.error(`\n${result.errors}\n`);
+
+  return 1;
+};
+
+exports.handler = async (argv) => {
+  verbose(argv.verbose);
+
+  const schema = yaml.load(
+    fs.readFileSync(join(__dirname, 'java-config-schema.yml'))
```

Comment:
It seems a bit problematic to merge this if it's hard-coded to the Java config schema.